plugin/zoom/lib/API/MeetingRegistrant.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+namespace Chamilo\PluginBundle\Zoom\API;
+
+use Exception;
+
+class MeetingRegistrant
```

Comment:
Codacy found an issue: [The class MeetingRegistrant has 17 fields. Consider redesigning MeetingRegistrant to keep the number of fields under 15.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)